spot_data_responses.go

```diff
@@ -181,5 +211,50 @@ func (r *SingleMarketStatistics) Parse(raw_response *http.Response) (*SingleMark
 		return nil, err
 	}
 	defer raw_response.Body.Close()
-	return r, err
+	return r, nil
+}
+
+func (r *AllMarketStatistics) Parse(raw_response *http.Response) (*AllMarketStatistics, error) {
```

Comment:
Do not write all methods of "different types" under each other,
Instead, Write methods under their own type.

Do this:
```go
type sth1 struct{}
func (s sth1) Parse1() {}
func (s sth1) Parse2() {}

type sth2 struct{}
func (s sth2) Parse1() {}
func (s sth2) Parse2() {}
```